### ISSUE TITLE
Process with an id of is not running errors on 0.9.9.8

### ISSUE THREAD

**Issue by cleung2010** [chocolatey/choco#346]
Encountered the same issue as chocolatey/chocolatey#603 on 0.9.9.8 and (and I get this same error on 0.9.8.28 too)

I have triage it down to `Get-ChocolateyUnzip.ps1` https://github.com/chocolatey/choco/blob/master/src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1#L95

Further verified by running 7zip's GUI (so that there is a 7zip process running). The package installs fine when the GUI is opened, but when I close the program and there is no 7zip process the installation fails.

The same package also works fine on 0.9.8.27

```
2015-06-29 15:30:22,809 [INFO ] - ============================================================
2015-06-29 15:30:22,809 [INFO ] - Chocolatey v0.9.9.8
2015-06-29 15:30:22,825 [DEBUG] - Chocolatey is running on Windows v 6.1.7601.65536
2015-06-29 15:30:22,825 [DEBUG] - Attempting to delete file "C:/ProgramData/chocolatey/choco.exe.old".
2015-06-29 15:30:22,825 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\choco.exe.old".
2015-06-29 15:30:22,825 [DEBUG] - Command line: "C:\ProgramData\chocolatey\choco.exe" install consul -version 0.5.2.2015052201 -dvy -source C:/Users/vagrant/AppData/Local/Temp
2015-06-29 15:30:22,825 [DEBUG] - Received arguments: install consul -version 0.5.2.2015052201 -dvy -source C:/Users/vagrant/AppData/Local/Temp
2015-06-29 15:30:22,872 [DEBUG] - 
NOTE: Hiding sensitive configuration data! Please double and triple 
 check to be sure no sensitive data is shown, especially if copying 
 output to a gist for review.
2015-06-29 15:30:22,887 [DEBUG] - Configuration: CommandName='install'|
CacheLocation='C:\Users\vagrant\AppData\Local\Temp'|
ContainsLegacyPackageInstalls='True'|
CommandExecutionTimeoutSeconds='2700'|
Sources='C:/Users/vagrant/AppData/Local/Temp'|Debug='True'|
Verbose='True'|Force='False'|Noop='False'|HelpRequested='False'|
RegularOutput='True'|PromptForConfirmation='False'|
AcceptLicense='True'|
AllowUnofficialBuild='False'|Input='consul'|Version='0.5.2.2015052201'|
AllVersions='False'|SkipPackageInstallProvider='False'|
PackageNames='consul'|Prerelease='False'|ForceX86='False'|
OverrideArguments='False'|NotSilent='False'|IgnoreDependencies='False'|
AllowMultipleVersions='False'|ForceDependencies='False'|
Information.PlatformType='Windows'|
Information.PlatformVersion='6.1.7601.65536'|
Information.PlatformName='Windows Server 2008 R2'|
Information.ChocolateyVersion='0.9.9.8'|
Information.ChocolateyProductVersion='0.9.9.8'|
Information.FullName='choco, Version=0.9.9.8, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|

Information.Is64Bit='True'|Information.IsInteractive='False'|
Information.IsUserAdministrator='True'|
Information.IsProcessElevated='True'|Features.AutoUninstaller='False'|
Features.CheckSumFiles='True'|Features.FailOnAutoUninstaller='False'|
ListCommand.LocalOnly='False'|
ListCommand.IncludeRegistryPrograms='False'|
UpgradeCommand.FailOnUnfound='False'|
UpgradeCommand.FailOnNotInstalled='False'|
UpgradeCommand.NotifyOnlyAvailableUpgrades='False'|
NewCommand.AutomaticPackage='False'|SourceCommand.Command='unknown'|
FeatureCommand.Command='unknown'|PushCommand.TimeoutInSeconds='0'|
PinCommand.Command='unknown'|
2015-06-29 15:30:22,887 [DEBUG] - _ Chocolatey:ChocolateyInstallCommand - Normal Run Mode _
2015-06-29 15:30:22,918 [INFO ] - Installing the following packages:
2015-06-29 15:30:22,918 [INFO ] - consul
2015-06-29 15:30:22,918 [INFO ] - By installing you accept licenses for the packages.
2015-06-29 15:30:23,231 [INFO ] - [NuGet] Installing 'consul 0.5.2.2015052201'.
2015-06-29 15:30:23,247 [DEBUG] - [NuGet] Added file '0.5.2_SHA256SUMS' to folder 'consul\binaries'.
2015-06-29 15:30:23,247 [DEBUG] - [NuGet] Added file '0.5.2_web_ui.zip' to folder 'consul\binaries'.
2015-06-29 15:30:23,247 [DEBUG] - [NuGet] Added file '0.5.2_windows_386.zip' to folder 'consul\binaries'.
2015-06-29 15:30:23,262 [DEBUG] - [NuGet] Added file 'nssm-2.24.zip' to folder 'consul\binaries'.
2015-06-29 15:30:23,262 [DEBUG] - [NuGet] Added file 'chocolateyInstall.ps1' to folder 'consul\tools'.
2015-06-29 15:30:23,262 [DEBUG] - [NuGet] Added file 'chocolateyUninstall.ps1' to folder 'consul\tools'.
2015-06-29 15:30:23,262 [DEBUG] - [NuGet] Added file 'consul.nupkg' to folder 'consul'.
2015-06-29 15:30:23,262 [INFO ] - [NuGet] Successfully installed 'consul 0.5.2.2015052201'.
2015-06-29 15:30:23,262 [INFO ] - 
consul v0.5.2.2015052201
2015-06-29 15:30:23,293 [DEBUG] - Contents of 'C:\ProgramData\chocolatey\lib\consul\tools\chocolateyInstall.ps1':
2015-06-29 15:30:23,309 [DEBUG] - try {
  $binariesPath = $(Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) "..\binaries\")
  $toolsPath = (Split-Path -Parent $MyInvocation.MyCommand.Definition)

  # NSSM related variables
  $nssmVersion = '2.24'
  $nssmSourcePath = $(Join-Path $binariesPath "nssm-$nssmVersion.zip")

  # Consul related variables
  $consulVersion = '0.5.2'
  $sourcePath = $(Join-Path $binariesPath "$($consulVersion)_windows_386.zip")
  $sourcePathUI = $(Join-Path $binariesPath "$($consulVersion)_web_ui.zip")

  # Install NSSM locally within consul
  Get-ChocolateyUnzip $nssmSourcePath $toolsPath

  $folderToIgnore = 'win32'
  $forderToRun = 'win64'

  if (Get-ProcessorBits 32) {
    $folderToIgnore = 'win64'
    $forderToRun = 'win32'
  }

  Set-Content -Path ($toolsPath + "\nssm-$nssmVersion\$folderToIgnore\nssm.exe.ignore") -Value $null
  $nssmBinPath = ($toolsPath + "\nssm-$nssmVersion\$forderToRun\nssm.exe")

  # Unzip and move Consul
  Get-ChocolateyUnzip  $sourcePath "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
  Get-ChocolateyUnzip  $sourcePathUI "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"

  Write-Host "Creating $env:PROGRAMDATA\consul\logs"
  New-Item -ItemType directory -Path "$env:PROGRAMDATA\consul\logs" -ErrorAction SilentlyContinue | Out-Null
  Write-Host "Creating $env:PROGRAMDATA\consul\config"
  New-Item -ItemType directory -Path "$env:PROGRAMDATA\consul\config" -ErrorAction SilentlyContinue | Out-Null

  # Create event log source
  # User -Force to avoid "A key at this path already exists" exception. Overwrite not an issue since key is not further modified
  $registryPath = 'HKLM:\SYSTEM\CurrentControlSet\services\eventlog\Application'
  New-Item -Path $registryPath -Name consul -Force | Out-Null
  # Set EventMessageFile value
  Set-ItemProperty $registryPath\consul EventMessageFile "C:\Windows\Microsoft.NET\Framework64\v2.0.50727\EventLogMessages.dll" | Out-Null


  #Uninstall service if it already exists. Stops the service first if it's running
  $service = Get-Service "consul" -ErrorAction SilentlyContinue
  if ($service) {
    Write-Host "Uninstalling existing service"
    if ($service.Status -eq "Running") {
      Write-Host "Stopping consul process ..."
      net stop consul | Out-Null
    }

    $service = Get-WmiObject -Class Win32_Service -Filter "Name='consul'"
    $service.delete() | Out-Null
  }

  Write-Host "Installing the consul service"
  # Install the service
  & $nssmBinPath install consul $(Join-Path $toolsPath "consul.exe") agent -config-dir=%PROGRAMDATA%\consul\config -data-dir=%PROGRAMDATA%\consul\data | Out-Null
  & $nssmBinPath set consul AppEnvironmentExtra GOMAXPROCS=$env:NUMBER_OF_PROCESSORS | Out-Null
  & $nssmBinPath set consul ObjectName NetworkService | Out-Null
  & $nssmBinPath set consul AppStdout "$env:PROGRAMDATA\consul\logs\consul-output.log" | Out-Null
  & $nssmBinPath set consul AppStderr "$env:PROGRAMDATA\consul\logs\consul-error.log" | Out-Null
  & $nssmBinPath set consul AppRotateBytes 10485760 | Out-Null
  & $nssmBinPath set consul AppRotateFiles 1 | Out-Null
  & $nssmBinPath set consul AppRotateOnline 1 | Out-Null

  # Restart service on failure natively via Windows sc. There is a memory leak if service restart is performed via NSSM
  # The NSSM configuration will set the default behavior of NSSM to stop the service if
  # consul fails (for example, unable to resolve cluster) and end the nssm.exe and consul.exe process.
  # The sc configuration will set Recovery under the Consul service properties such that a new instance will be started on failure,
  # spawning new nssm.exe and consul.exe processes. In short, nothing changed from a functionality perspective (the service will
  # still attempt to restart on failure) but this method kills the nssm.exe process thus avoiding memory hog.
  & $nssmBinPath set consul AppExit Default Exit | Out-Null
  cmd.exe /c "sc failure consul reset= 0 actions= restart/60000" | Out-Null

  Write-ChocolateySuccess 'consul'
} catch {
  Write-ChocolateyFailure 'consul' $($_.Exception.Message)
  throw
}

2015-06-29 15:30:23,309 [DEBUG] - Calling command ['"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -NoLogo -ExecutionPolicy Bypass -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = '';[System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; & import-module -name 'C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1'; & 'C:\ProgramData\chocolatey\helpers\chocolateyScriptRunner.ps1' -packageScript 'C:\ProgramData\chocolatey\lib\consul\tools\chocolateyInstall.ps1' -installArguments '' -packageParameters ''"']
2015-06-29 15:30:23,950 [ERROR] -  Process with an Id of 2984 is not running.
2015-06-29 15:30:23,950 [DEBUG] -  DEBUG: Posh version is 2.0
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-BinRoot'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-ChecksumValid'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-ChocolateyUnzip'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-ChocolateyWebFile'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-EnvironmentVariable'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-EnvironmentVariableNames'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-FtpFile'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-ProcessorBits'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-UACEnabled'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-VirusCheckValid'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-WebFile'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Get-WebHeaders'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-BinFile'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyDesktopLink'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyEnvironmentVariable'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyExplorerMenuItem'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyFileAssociation'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyInstallPackage'.
2015-06-29 15:30:23,950 [ERROR] -  At C:\ProgramData\chocolatey\helpers\functions\Write-ChocolateyFailure.ps1:24 c
2015-06-29 15:30:23,950 [ERROR] -  har:8
2015-06-29 15:30:23,950 [ERROR] -  +   throw <<<<  "$failureMessage"
2015-06-29 15:30:23,950 [ERROR] -      + CategoryInfo          : OperationStopped: (Process with an...is not runn 
2015-06-29 15:30:23,950 [ERROR] -     ing.:String) [], RuntimeException
2015-06-29 15:30:23,950 [ERROR] -      + FullyQualifiedErrorId : Process with an Id of 2984 is not running.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyPackage'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyPath'.
2015-06-29 15:30:23,950 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyPinnedTaskBarItem'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyPowershellCommand'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyShortcut'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyVsixPackage'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Install-Vsix'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Install-ChocolateyZipPackage'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Set-EnvironmentVariable'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Start-ChocolateyProcessAsAdmin'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Test-ProcessAdminRights'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Uninstall-BinFile'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Uninstall-ChocolateyPackage'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'UnInstall-ChocolateyZipPackage'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Update-SessionEnvironment'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Write-ChocolateyFailure'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Write-ChocolateySuccess'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting function 'Write-FileUpdateLog'.
2015-06-29 15:30:23,965 [INFO ] -  VERBOSE: Exporting alias 'Generate-BinFile'.
2015-06-29 15:30:23,997 [INFO ] -  VERBOSE: Exporting alias 'Add-BinFile'.
2015-06-29 15:30:23,997 [INFO ] -  VERBOSE: Exporting alias 'Remove-BinFile'.
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: Running 'Get-ChocolateyUnzip' with 
2015-06-29 15:30:23,997 [INFO ] -  fileFullPath:'C:\ProgramData\chocolatey\lib\consul\tools\..\binaries\nssm-2.24.
2015-06-29 15:30:23,997 [INFO ] -  zip'', destination: 'C:\ProgramData\chocolatey\lib\consul\tools', 
2015-06-29 15:30:23,997 [INFO ] -  specificFolder: '`, packageName: ''
2015-06-29 15:30:23,997 [INFO ] -  Extracting C:\ProgramData\chocolatey\lib\consul\tools\..\binaries\nssm-2.24.zip to C:\ProgramData\chocolatey\lib\consul\tools...
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: 7zip found at 'C:\ProgramData\chocolatey\tools\7za.exe'
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: calling 7za directly
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: Calling 'C:\ProgramData\chocolatey\tools\7za.exe x -aoa 
2015-06-29 15:30:23,997 [INFO ] -  -o"C:\ProgramData\chocolatey\lib\consul\tools" -y 
2015-06-29 15:30:23,997 [INFO ] -  "C:\ProgramData\chocolatey\lib\consul\tools\..\binaries\nssm-2.24.zip"'
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: 7za exit code: 0
2015-06-29 15:30:23,997 [INFO ] -  C:\ProgramData\chocolatey\lib\consul\tools
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: Running 'Get-ProcessorBits'
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: Running 'Get-ChocolateyUnzip' with 
2015-06-29 15:30:23,997 [INFO ] -  fileFullPath:'C:\ProgramData\chocolatey\lib\consul\tools\..\binaries\0.5.2_wind
2015-06-29 15:30:23,997 [INFO ] -  ows_386.zip'', destination: 'C:\ProgramData\chocolatey\lib\consul\tools', 
2015-06-29 15:30:23,997 [INFO ] -  specificFolder: '`, packageName: ''
2015-06-29 15:30:23,997 [INFO ] -  Extracting C:\ProgramData\chocolatey\lib\consul\tools\..\binaries\0.5.2_windows_386.zip to C:\ProgramData\chocolatey\lib\consul\tools...
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: 7zip found at 'C:\ProgramData\chocolatey\tools\7za.exe'
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: calling 7za directly
2015-06-29 15:30:23,997 [DEBUG] -  DEBUG: Calling 'C:\ProgramData\chocolatey\tools\7za.exe x -aoa 
2015-06-29 15:30:23,997 [INFO ] -  -o"C:\ProgramData\chocolatey\lib\consul\tools" -y 
2015-06-29 15:30:23,997 [INFO ] -  "C:\ProgramData\chocolatey\lib\consul\tools\..\binaries\0.5.2_windows_386.zip"'
2015-06-29 15:30:24,012 [WARN ] -  WARNING: Write-ChocolateyFailure is deprecated. If you are the package 
2015-06-29 15:30:24,012 [INFO ] -  maintainer, please use 'throw $_.Exception' instead.
2015-06-29 15:30:24,012 [DEBUG] - Command ['"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -NoLogo -ExecutionPolicy Bypass -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = '';[System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; & import-module -name 'C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1'; & 'C:\ProgramData\chocolatey\helpers\chocolateyScriptRunner.ps1' -packageScript 'C:\ProgramData\chocolatey\lib\consul\tools\chocolateyInstall.ps1' -installArguments '' -packageParameters ''"'] exited with '1'
2015-06-29 15:30:24,012 [DEBUG] - Calling command ['"shutdown" /a']
2015-06-29 15:30:24,028 [DEBUG] - Command ['"shutdown" /a'] exited with '1116'
2015-06-29 15:30:24,043 [DEBUG] - Capturing package files in 'C:\ProgramData\chocolatey\lib\consul'
2015-06-29 15:30:24,075 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\consul.nupkg'
  with checksum '66A13E5A10A7E7CA1B20EFF2EA2208AC'
2015-06-29 15:30:24,075 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\binaries\0.5.2_SHA256SUMS'
  with checksum '63289EDFB26AFB48E0E4CD8110629D01'
2015-06-29 15:30:24,075 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\binaries\0.5.2_web_ui.zip'
  with checksum 'EB98BA602BC7E177333EB2E520881F4F'
2015-06-29 15:30:24,090 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\binaries\0.5.2_windows_386.zip'
  with checksum 'D5DE018E601DB10850F89C1E81559D4A'
2015-06-29 15:30:24,090 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\binaries\nssm-2.24.zip'
  with checksum 'B2EDD0E4A7A7BE9D157C0DA0EF65B1BC'
2015-06-29 15:30:24,090 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\chocolateyInstall.ps1'
  with checksum '60F66EE4FE81706134EB5E8E2072EA65'
2015-06-29 15:30:24,090 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\chocolateyUninstall.ps1'
  with checksum 'D3507DFEF5209948F073295CF6484F6B'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\consul.exe'
  with checksum '974432CDFFAF53462789D8A87ED4D518'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\ChangeLog.txt'
  with checksum '205769831FA302018FB26845D3835962'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\README.txt'
  with checksum 'ED2BC805600C380E85FE240E715ABB81'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\account.cpp'
  with checksum 'B432CC0EDDB92622E68C16944EF65E98'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\account.h'
  with checksum 'C1B1DCFDCE66CB24B458B0F34278C8DB'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\console.cpp'
  with checksum '2B504E046DC126E68BFC90056B2C9E60'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\console.h'
  with checksum '9E242CCE5F18C22854F7E417171E5E67'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\env.cpp'
  with checksum 'F49F53D2A1527B35258B33773D048B9C'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\env.h'
  with checksum 'F3C5C5EC2AEAFA00FF4C2FBE5622521C'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\event.cpp'
  with checksum '53CA1240B593D6277E58A9D3334B537C'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\event.h'
  with checksum 'BD2D50FA5FBBB97285F9984E6CD1E4D4'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\gui.cpp'
  with checksum '87A2C4762F8F544056F223315F6F8A90'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\gui.h'
  with checksum 'A015F901E2ECE542EC8DA0D62A2DEC91'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\imports.cpp'
  with checksum 'B3841F61DF8EAAE6503C4535FFFA2678'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\imports.h'
  with checksum 'BEA805FC9BD24E29F911B4C8AF2E1C67'
2015-06-29 15:30:24,137 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\io.cpp'
  with checksum '4A0F774542B30B8D2F89B16A52F4E180'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\io.h'
  with checksum 'D38837B657FD6110F6DFCC464CB5A28A'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\messages.mc'
  with checksum '8CE6DC904ADF43A36CC1874B56FE10E5'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\nssm.cpp'
  with checksum '4C6D4F21F01D8EA96856B9BE08819006'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\nssm.h'
  with checksum '489522C3667DE3BAF5F28A66DD684005'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\nssm.ico'
  with checksum '4288408C2D7F43FBD9D920C8319F1956'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\nssm.rc'
  with checksum 'FFACD065D645999917FF340D7C7E9EE0'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\nssm.sln'
  with checksum 'F779189B1586DFA1D64F9279AD8E23EC'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\nssm.vcproj'
  with checksum '6E6FABD941F67804A55F5D9EE21A7E11'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\process.cpp'
  with checksum 'E58E6825766F138869E72B66085C0758'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\process.h'
  with checksum '7D2B1A9335864BB2C334273FE40CF732'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\registry.cpp'
  with checksum '445B79BFC0A559C5CF4C048C54218566'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\registry.h'
  with checksum '12766774BADE41953CA460AE72168F1B'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\resource.h'
  with checksum 'A06FEE0B96D7CD81E1B0A9CB9E359396'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\service.cpp'
  with checksum '95FB7517F1606492A69D3B3DDEC9C01D'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\service.h'
  with checksum 'E317D08FBC98329115C47D8095190D83'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\settings.cpp'
  with checksum '762C0AC39328C797AECEF896A1319877'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\settings.h'
  with checksum '92BFCAF191385245DF8A907C7DD27E3E'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\src\version.cmd'
  with checksum '3E67B67B2ECB2D3042DFAA7D216883B1'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\win32\nssm.exe'
  with checksum 'D9EC6F3A3B2AC7CD5EEF07BD86E3EFBC'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\win32\nssm.exe.ignore'
  with checksum 'D41D8CD98F00B204E9800998ECF8427E'
2015-06-29 15:30:24,153 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\consul\tools\nssm-2.24\win64\nssm.exe'
  with checksum 'BECEAE2FDC4F7729A93E94AC2CCD78CC'
2015-06-29 15:30:24,153 [DEBUG] - Attempting to create directory "C:\ProgramData\chocolatey\.chocolatey".
2015-06-29 15:30:24,168 [DEBUG] - Attempting to create directory "C:\ProgramData\chocolatey\.chocolatey\consul.0.5.2.2015052201".
2015-06-29 15:30:24,215 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\consul.0.5.2.2015052201\.sxs".
2015-06-29 15:30:24,215 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\consul.0.5.2.2015052201\.pin".
2015-06-29 15:30:24,231 [ERROR] - The install of consul was NOT successful.
2015-06-29 15:30:24,231 [ERROR] - Error while running 'C:\ProgramData\chocolatey\lib\consul\tools\chocolateyInstall.ps1'.
 See log for details.
2015-06-29 15:30:24,231 [DEBUG] - Attempting to create directory "C:\ProgramData\chocolatey\lib-bad".
2015-06-29 15:30:24,231 [DEBUG] - Moving 'C:\ProgramData\chocolatey\lib\consul'
 to 'C:\ProgramData\chocolatey\lib-bad\consul'
2015-06-29 15:30:26,231 [WARN ] - 
Chocolatey installed 0/1 package(s). 1 package(s) failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
2015-06-29 15:30:26,231 [ERROR] - Failures:
2015-06-29 15:30:26,231 [ERROR] -  - consul
2015-06-29 15:30:26,231 [DEBUG] - Exiting with 1
```

**Comment by ferventcoder** [comment]
This commit first appears in 0.9.8.28:
https://github.com/chocolatey/chocolatey/commit/d6068c2fb8b6cb9d31a469cb8688de2f3d4dc473

**Comment by ferventcoder** [comment]
Dump the try catch in the package (and the write-success/write-failure) - that way we can see which line is triggering the issue. I'm thinking it might be the second call in a row to `Get-ChocolateyUnzip`.

**Comment by ferventcoder** [comment]
These commits also touched:
* https://github.com/chocolatey/choco/commit/6f6dec6b62a716b25bf19990e396a5560764438a
* https://github.com/chocolatey/choco/commit/ea6f1ff83c24ce0007c48dd19e33b874c97bc377

**Comment by ferventcoder** [comment]
ping @cleung2010 - have you had an opportunity to gather more information related to this?

**Comment by cleung2010** [comment]
Sorry, I have been quite busy lately. I'll try to do a test run with the suggested changes sometime this week.

**Comment by ferventcoder** [comment]
Thanks for the update! :)

**Comment by ferventcoder** [comment]
Closed by #400

**Comment by cleung2010** [comment]
Awesome! Any ETA on when it's going to be released? Sorry for bailing on dumping my more logs on this, but glad it's resolved now.
